test/functional/like_controller_test.rb

```diff
@@ -29,7 +29,7 @@ def teardown; end
     assert_response :success
 
     note = Node.find note.id
-    assert_equal @response.body, '1'
```

Comment:
why did u change this to an int? One of the tests is faling because it expects "1" and not 1.